boards/arm/mps2_an521/CMakeLists.txt

```diff
@@ -8,9 +8,11 @@ zephyr_library()
 zephyr_library_sources(pinmux.c)
 zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
 
-if(CONFIG_SOC_MPS2_AN521_CPU1)
+if(CONFIG_SOC_MPS2_AN521_CPU1 AND NOT CONFIG_OPENAMP)
```

Comment:
should this be more generic ?
Afterall, OpenAmp might not be the only sample that should disable the empty-cpu0.

Reply:
Well, it could. Although, only openAMP samples/applications/tests for Cortex-M are currently multi-core.
Honestly, I would not introduce a new abstract option, since we only use this, currently, in the MPS2 AN521 platform.

Reply:
@tejlmand, I thought about it and my feeling is that we are OK with the comment that is placed here, inline.
We could have a CONFIG_DUAL_CORE_APPLICATION option to instruct the build, here, _not_ to build an empty CPU0, but since openamp is the only in-tree subsystem that currently relates to multi-core applications, I would skip introducing it in favor of having the explanatory comment.